misc/build-registry-image.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e
+
+version=$(git descibe --exact 2>/dev/null || git rev-parse --short=1- HEAD)
```

Comment:
`entropic/registry:vX.X.X` or `entropic/registry:XXXXXXXXXX`. Since there's no Docker repo yet I can change this if need be.